src/components/messenger/list/conversation-list-panel/index.tsx

```diff
@@ -53,6 +53,30 @@ export class ConversationListPanel extends React.Component<Properties, State> {
     this.scrollContainerRef = React.createRef();
   }
 
+  componentDidMount() {
+    this.openMostRecentConversation();
+  }
+
+  openMostRecentConversation = () => {
```

Comment:
This won't be required, we shouldn't be handling this sort of functionality at component level. Please take a look at the saga where the logic lives for handling conversations. We already have a `openFirstConversation` saga function that handles the opening of the first conversation. If there is an issue, this is where you need to investigate. It would probably be worth looking into the room events, if they are being handled/received correctly to determine what conversation to open.

`src/store/channels/saga.ts` 
`src/lib/chat/matrix-client.ts`